apps/crn-frontend/src/discover/Routes.tsx

```diff
@@ -48,9 +50,14 @@ const Discover: FC<Record<string, never>> = () => {
             </Frame>
           </Route>
           <Route exact path={path + discover({}).tutorials.template}>
-            <Frame title="Tutorials">
-              <TutorialList />
-            </Frame>
+            <TutorialsPage
```

Comment:
Usually when we talk about pages its the header. Maybe this should be TutorialSearch?

Reply:
It occurs you should use the SearchFrame in the component higher than the useTutorials hook. I recently had to change this in a bunch of other places